federation/pkg/federation-controller/ingress/ingress_controller.go

```diff
@@ -830,14 +804,7 @@ func (ic *IngressController) reconcileIngress(ingress types.NamespacedName) {
 					return
 				}
 				if !baseLBStatusExists && clusterLBStatusExists {
-					lbstatusObj, lbErr := api.Scheme.DeepCopy(&clusterIngress.Status.LoadBalancer)
-					lbstatus, ok := lbstatusObj.(*v1.LoadBalancerStatus)
-					if lbErr != nil || !ok {
-						glog.Errorf("Internal error: Failed to clone LoadBalancerStatus of %q in cluster %q while attempting to update master loadbalancer ingress status, will try again later. error: %v, Object to be cloned: %v", ingress, cluster.Name, lbErr, lbstatusObj)
-						ic.deliverIngress(ingress, ic.ingressReviewDelay, true)
-						return
-					}
-					baseIngress.Status.LoadBalancer = *lbstatus
+					clusterIngress.Status.LoadBalancer.DeepCopyInto(&baseIngress.Status.LoadBalancer)
```

Comment:
Why is this better than `baseIngress.Status.LoadBalancer = *clusterIngress.Status.LoadBalancer.DeepCopy()`?